From 80e016cc4efef1b7f9abc73e209e83bcd3fa0c6f Mon Sep 17 00:00:00 2001
From: Rob Crittenden <rcritten@redhat.com>
Date: Tue, 17 Jul 2018 16:22:02 -0400
Subject: [PATCH] Disable the 10-iterate tests which randomly fail

Sometimes the dates are off just enough to cause the comparison
to fail.
---
 tests/Makefile.am | 3 ---
 1 file changed, 3 deletions(-)

diff --git a/tests/Makefile.am b/tests/Makefile.am
index f60e571..bbcd06e 100644
--- a/tests/Makefile.am
+++ b/tests/Makefile.am
@@ -62,8 +62,6 @@ EXTRA_DIST = \
 	008-certread/expected.out \
 	009-oiddict/run.sh \
 	009-oiddict/expected.out \
-	010-iterate/run.sh \
-	010-iterate/expected.out \
 	011-dbinit/expected.out \
 	011-dbinit/run.sh \
 	011-dbinit-dbm/expected.out \
@@ -166,7 +164,6 @@ subdirs = \
 	007-certsave \
 	008-certread \
 	009-oiddict \
-	010-iterate \
 	011-dbinit \
 	012-dbadd \
 	013-enckey \
-- 
2.13.6

